src/ioctls/vm.rs

```diff
@@ -609,12 +607,7 @@ mod tests {
         let kvm = Kvm::new().unwrap();
         assert!(kvm.check_extension(Cap::Irqchip));
         let vm = kvm.create_vm().unwrap();
-        if cfg!(any(target_arch = "x86", target_arch = "x86_64")) {
-            assert!(vm.create_irq_chip().is_ok());
-        } else if cfg!(any(target_arch = "arm", target_arch = "aarch64")) {
-            // On arm, we expect this to fail as the irq chip needs to be created after the vcpus.
```

Comment:
The tests are failing and I believe it is related to the above comment.

Reply:
Looking further, I think if you remove this hunk (test_create_irq_chip() changes) then the tests should pass.